src/plugins/data/public/index_patterns/index_patterns/index_patterns_api_client.ts

```diff
@@ -36,6 +36,11 @@ export class IndexPatternsApiClient implements IIndexPatternsApiClient {
         query,
       })
       .catch((resp: any) => {
+        // eslint-disable-next-line no-console
+        console.error('IndexPatternsApiClient ERROR');
+        // eslint-disable-next-line no-console
+        console.error('IndexPatternsApiClient ERROR DEBUG', JSON.stringify(resp));
+
```

Comment:
@mattkime this is the latest add-on to my console logging that I would search for in the logs